pkg/apiserver/apiserver.go

```diff
@@ -78,6 +78,10 @@ type Mux interface {
 type APIGroupVersion struct {
 	Storage map[string]rest.Storage
 
+	// Root is the APIPrefix under which this is being served.  It can also be the APIPrefix/APIGroup that is being served
```

Comment:
Thank you @deads2k. We use "group/version" in many places in #14156 for fast implementation. We should gradually split them into separate fields where we can.